### ISSUE TITLE
Возможное лекарство

### ISSUE THREAD

**Issue by greenberg3** [Maxx53/SCMBot#496]
1. Макс, пофикси вкладку "новые поступления"
Запрос на новые поступления в JSON 
http://steamcommunity.com/market/recent?country=RU&language=russian&currency=(ну тут валюта, ты знаешь)
2. Как работают хосты? я по ip не могу зайти на сайт
пишет:
Invalid URL

The requested URL "/", is invalid.
Reference #9.98643e17.1445572390.113cf1
3. Найден странный баг, если удалить куки, то бан снимается. но приходится авторизоваться в Guard'e, сделай один билд где куки удаляются и бот заходит снова по логину и паролю, нужно протестировать. 

P.s пробуйте тестировать без файла hosts

**Comment by vzapas** [comment]
Странно, как ввели баны, сразу пробовал метод с удалением куков и бан оставался висеть

**Comment by rnd0m** [comment]
какие именно куки? разве при удалении кукисов не слетает стимгвард (новое устройство)? или вы про какие-то другие речь ведете?

**Comment by Maxx53** [comment]
пф.. могу предложить не какое-то там "возможное" лекарство, а что ни на есть реальное:

**Comment by Igzibed** [comment]
@greenberg3 попробовал сделать как ты говоришь, т.е. удаляю cookies.dat в папке с программой, перезапускаю, вбиваю код steam guard, бан остается

**Comment by Maxx53** [comment]
пф.. могу предложить не какое-то там "возможное" лекарство, а самое что ни на есть реальное:
![Image](http://bloknot-stavropol.ru/thumb/600x0xcut/upload/iblock/d9c/boyaryshnik.jpg)

**Comment by X4ndrius** [comment]
@Maxx53 Дайте парочку =)

**Comment by nickden** [comment]
заметил что, в браузере вкладка новые работает даже при бане

**Comment by nikita971** [comment]
Да, новые поступления неплохо было бы прочекать, они возможно без бана будут идти, а вот про куки реально бред, попробуй вообще не с аккаунта зайти на страницу предмета будет то же самое:D

**Comment by amokrushin** [comment]
"2. Как работают хосты? я по ip не могу зайти на сайт пишет: Invalid URL"
На одном хосте может быть множество сайтов, чтобы хост знал к какому сайту обращаются передается заголовок запроса, в данном случае это Host:steamcommunity.com, при переходе в браузере по ip данный заголовок не передается

**Comment by greenberg3** [comment]
дело в том что, выводится бан, что много запросов, удаляю куки, и бан слетает. не могу понять в чем дело, после входа, тп в рабочем состоянии.

**Comment by greenberg3** [comment]
Просто он работал через раз, если вы изначально бан получили не по хостам, то снять реально.
в браузере не все очищается. там на поддоменах куки еще.

**Comment by greenberg3** [comment]
и тут есть еще прикол, есть дополнение steam inventory helper
там можно добавлять в список (желаний, наверно) предметы, и они раз в 10 секунд обновляют цену. и оно работает при бане.
![default](https://cloud.githubusercontent.com/assets/13793796/10699233/088f2704-79be-11e5-8bc5-7df824b8dc2a.JPG)

**Comment by greenberg3** [comment]
![default](https://cloud.githubusercontent.com/assets/13793796/10699309/74c86912-79be-11e5-8aeb-ed62b65f96ae.JPG)
Вот такие вот запросы, данная страница работает при бане. но это лишь JSON с количеством и минимальной ценой.

**Comment by greenberg3** [comment]
Можно костыльнуть, сканить это страницу, и смотреть уже реальную страницу (при обнуружении минимальной цены)

**Comment by nikita971** [comment]
не надо этой всей херней заниматься, сделаете, школота опять накачает этого, вырастет конкуренция и потом будете плакаться почему стим ввел капчу на скупку или бан на 7 дней как в дотке:/ если хотите что-то делать, то пожалуйста не пишите это в паблик и не скидывайте в паблик ваши идеи, иначе всему придет конец, он уже практически пришел.

**Comment by MazaFaker** [comment]
@nikita971 что ты несешь ? причем тут вообще школота, о какой конкуренции идет речь если в такой ситуации нечем просто конкурировать. Свои "умные" мысли лучше оставь при себе !

**Comment by nikita971** [comment]
Такую ситуацию создала как раз школота, и сейчас есть чем конкурировать, просто тебе нечем согласен, но другие люди в теме. Пройдет месяц, в прогу пихнут что-нибудь, что обходит бан и снова начнется:/...

**Comment by MazaFaker** [comment]
@nikita971 Поверь, я в курсе каким способом ты сейчас ботишь и "другие", не строй из себя дохрена умного, но этот способ никак не подходит мне, так же другим подобным мне ботоводам

**Comment by ninjagendalf** [comment]
@MazaFaker тоже много вещей ботишь?)
